live/core/README.md

```diff
@@ -102,16 +102,44 @@ vpc_id=$(aws ssm get-parameter --name "$vpc_id_parameter_name" --query 'Paramete
 echo "VPC ID: $vpc_id"
 ```
 
-### Connecting to the Bastion Host
+## Connecting to the Bastion Host
 
-To establish a connection with the bastion host, use the following commands:
+To establish a secure connection with the bastion host, follow these steps:
+
+### 1. Obtain Required Information
+
+First, you need to gather some essential information:
+
+- Bastion SSH Parameter Name
+- SSH Key File Name
+- Bastion Instance ID
+
+You can retrieve these values using Terraform:
 
 ```bash
-# Retrieve the SSH command from the Terraform output
-ssh_command=$(terraform output -json | jq -r '.bastion_ssh_command.value')
+bastion_ssh_parameter_name=$(terraform output -json | jq -r '.ssm_parameter_bastion_ssh_key.value')
+ssh_key_file_name=$(terraform output -json | jq -r '.bastion_ssh_key_name.value') 
+bastion_instance_id=$(terraform output -json | jq -r '.bastion_instance_id.value')
+```
+
+### 2. Generate .pem file with the ssh key
 
-# Execute the SSH command
-eval $ssh_command
+```bash
+ssh_private_key=$(aws ssm get-parameter --name "$bastion_ssh_parameter_name" --with-decryption --query 'Parameter.Value' --output text)
+echo "$ssh_private_key" > "$ssh_key_file_name"
```

Comment:
you can prevent storing it in a variable. Doing something like this should work:

```sh
aws ssm get-parameter --name "$bastion_ssh_parameter_name" --with-decryption --query 'Parameter.Value' --output text > /tmp/something.pem
```